units/systemd-timesyncd.service.in

```diff
@@ -36,6 +36,7 @@ RestrictNamespaces=yes
 RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
 SystemCallFilter=~@cpu-emulation @debug @keyring @module @mount @obsolete @raw-io
 SystemCallArchitectures=native
+ReadWritePaths=/var/lib/systemd
```

Comment:
Well, if timesyncd is started before that file exists, then we can't create such a bind mount, that's why I specified the directory containing that file instead of the file itself...